stories/__tests__/SQFormIconButton.stories.test.js

```diff
@@ -68,7 +68,11 @@ describe('SQFormIconButton Tests', () => {
       const svg = within(iconButton).getByTitle(/Form Submission/i);
 
       expect(iconButton).toBeInTheDocument();
-      expect(svg).toHaveStyle('color: var(--color-palmLeaf)');
+      expect(svg).toHaveStyle({
+        color: 'var(--color-teal)',
+        width: '2.5rem',
+        height: '2.5rem'
+      });
```

Comment:
@codyarose @laurelbean Is this testing implementation details? Thoughts?

Reply:
we had to add all styles because it requires all elements of style
![image](https://user-images.githubusercontent.com/76514364/120523749-b965f900-c3ef-11eb-9cab-95aecebefdbd.png)
https://github.com/testing-library/jest-dom#tohavestyle

Reply:
@SeanGroff Yeah, this is all implementation details. If we need to test that the color is correct, then we pretty much have to test implementation details.

However, I have a feeling that there's a better way to do this, but I need to do some research on it. Essentially I'm thinking we can get the css-in-js value from `makeStyles` then snapshot test that.

Reply:
https://jestjs.io/docs/snapshot-testing

This looks like the route we'd want to take to properly test this.
However, I don't want to hold this up. I'm going to merge this for now and create a new issue to explorer snapshot testing situations like this where we want to ensure the correct styles apply based on a components props.